internal/v1/api.yaml

```diff
@@ -395,7 +395,6 @@ components:
       required:
         - architecture
         - image_type
-        - upload_request
```

Comment:
Upload request is required, no?

Reply:
I should've added it to the description, this is intended to address #368 , there are cases in which upload request is not necessary and can be inferred from the image type